Dockerfile.template

```diff
@@ -0,0 +1,132 @@
+ARG ARCH=%%BALENA_ARCH%%
```

Comment:
Should we wait to add this dockerfile when we have the actions ready? Otherwise we'll have two dockerfiles for a while

Reply:
Don't we want the compose file so we can build the Supervisor on balenaCloud ? I don't understand how we'll need just the compose but not the dockerfile to build a release since we're adding the compose to correctly build the Supervisor (use the right service name).

Reply:
We are not building the supervisor on balenaCloud yet, not until we can have the CI do it. We can still deploy in the same way, but now we need to make sure the docker-compose is on the local folder when we do manual deploy, as described [here](https://www.balena.io/docs/reference/balena-cli/#deploy-appname-image)